src/components/imageApprovalModal/index.tsx

```diff
@@ -0,0 +1,226 @@
+    setOpen(false);
+    props.onClose();
+  };
+
+  const openRejectionTextBox = () => {
+    setIsRejectionTextOpen(true);
+  };
+
+  const handleNextSubmission = () => {
+    const currentIndex = allData.findIndex(
+      (siteData) => siteData.key === data?.key,
+    );
+    const nextIndex = (currentIndex + 1) % allData.length;
+    const nextImage = allData[nextIndex];
+    setData(nextImage);
+  };
+
+  const footer = (
+    <Space style={{ width: '100%', justifyContent: 'space-between' }}>
+      <Button style={{ color: 'gray' }} onClick={close}>
+        <LeftOutlined /> Back to List
+      </Button>
+      <Button style={{ color: 'gray' }} onClick={handleNextSubmission}>
+        Next Submission <RightOutlined />
+      </Button>
+    </Space>
+  );
+
+  const rejectionReasonStyle = {
+    background: 'rgb(239, 239, 239)',
+    borderRadius: '5px',
+    marginTop: '5%',
+    padding: '3%',
+    paddingRight: '10%',
+  };
+
+  const StatusHeader = () => {
+    return (
+      <>
+        <h1 style={{ color: 'black', marginBottom: '-8px' }}>Status</h1>
+        <div
+          style={{
+            display: 'flex',
+            width: '100%',
+            backgroundColor: 'rgb(232, 240, 215)',
+            padding: '2%',
+            borderRadius: '5px',
+          }}
+        >
+          <Button
+            type="primary"
+            style={{ flex: 1, marginRight: '8px', color: 'black' }}
+            onClick={onClickAccept}
+          >
+            Accept
+          </Button>
+          <Button
+            type="primary"
+            style={{ flex: 1, color: 'black' }}
+            onClick={openRejectionTextBox}
+          >
+            Reject
+          </Button>
+        </div>
+      </>
+    );
+  };
+
+  if (!data) {
+    return null;
+  }
+
+  const treeSummaryTextStyle: CSSProperties = {
+    whiteSpace: 'nowrap', // Prevent text from wrapping
+  };
+
+  async function onClickReject() {
+    const toReject: Promise<void>[] = [];
+    if (!data) {
+      return null;
+    }
+    toReject.push(protectedApiClient.rejectImage(data.key, rejectionReason));
+    Promise.all(toReject).then(() => {
+      props.setApprovedOrRejectedImageIds((prevIds) => [...prevIds, data.key]);
+      close();
+    });
+  }
+
+  async function onClickAccept() {
+    const toApprove: Promise<void>[] = [];
+    if (!data) {
+      return null;
+    }
+    toApprove.push(protectedApiClient.approveImage(data.key));
+    Promise.all(toApprove).then(() => {
+      props.setApprovedOrRejectedImageIds((prevIds) => [...prevIds, data.key]);
+      close();
+    });
+  }
+
+  function treeSummaryLine(lineName: string, lineItem: number | string) {
+    return (
+      <>
+        <Col span={12}>
+          <b>
+            <p style={treeSummaryTextStyle}>{lineName}</p>
+          </b>
+        </Col>
+        <Col span={12}>
+          <p style={treeSummaryTextStyle}>{lineItem}</p>
+        </Col>
+      </>
+    );
+  }
+
+  const TreeSummaryDisplay = () => {
```

Comment:
⛏️ Right now, if someone selects an image on the list view and then switches to the grid view, it's not super clear which ones are selected:

![image](https://github.com/Code-4-Community/speak-for-the-trees-frontend/assets/68914997/af6bc0b1-0028-436a-9e8e-15432b380f9e)

Could we add some visuals to show which images are currently selected? Some things I'm thinking are:
- green border around the image
- something like in the figma where a checkmark is overlaid: (although I worry a little it might get read as "approved" instead of "selected")
![image](https://github.com/Code-4-Community/speak-for-the-trees-frontend/assets/68914997/533bdcb5-e831-47de-becc-d88457968e21)

though if you think of something you feel is better, definitely go for it!